### PR TITLE
PLATFORM-703: added php7 to travis build config

### PR DESCRIPTION
**Pull request by n1te1337**
@bigcommerce/platform

### CODE REVIEW

**Review by cwalsh — APPROVED** [review]
:+1: :heart:

**Comment by n1te1337** [comment]
@cwalsh  I can't seem to see the merge button, can you?

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
+
 php:
-  - 5.5
-  - 5.4
-  - 5.3
+  - 5.6
+  - 7.0
+
+install:
+  - composer install -o
```
Comment (petrkotek) [inline .travis.yml]: is `-o` necessary?

**Comment by cwalsh** [comment]
@n1te1337 You should see the button now. Not merging myself until you've seen Petr's comment though.

File: .travis.yml (hunk 1 above)
Comment (rayward) [inline .travis.yml]: Actually yeh I remember we don't use this for travis builds now, to help catch bad file/class naming which causes autoload issues.

Comment (n1te1337) [inline .travis.yml]: removed `-o`

**Pull request merged**